common/config/rush/npm-shrinkwrap.json

```diff
@@ -5203,6 +5207,549 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
+    "fsevents": {
```

Comment:
I don't know where this huge addition is coming from...

Reply:
Did you run the shrinkwrap command on macOS? That's the only place I've seen it added. I always manually delete the `fsevents` object after generating the shrinkwrap file.

Reply:
I ran the command on a PC. But it's good to know that it's something unwanted that does come up occasionally!  I'll go ahead and manually delete it, too.